include/cantera/kinetics/ChebyshevRate.h

```diff
@@ -242,19 +215,6 @@ class ChebyshevRate final : public ReactionRate
         return m_coeffs.nRows();
     }
 
-    //! Access the ChebyshevRate coefficients.
-    /*!
-     *  \f$ \alpha_{t,p} = \mathrm{coeffs}[N_P*t + p] \f$ where
-     *  \f$ 0 <= t < N_T \f$ and \f$ 0 <= p < N_P \f$.
-     *
```

Comment:
The new version uses a 2D array rather than the flattened coefficient vector, so it's not an exact correspondence. I believe the docstring of `data()` is adequate as it is.